.circleci/config.yml

```diff
@@ -131,11 +155,13 @@ workflows:
       - check_compatibility:
           python_version: "3.11"
           name: check-compatibility-3.11
-  
+
   pr-requirements:
     jobs:
       - black:
           python-version: "3.8.12"
+      - pylint:
+          python-version: "3.8.12"
```

Comment:
Something to be aware of, most of the subnets are actually building using python 3.9>, so we might want our code reflect that

Reply:
The most important change between typehints, >python 3.9 typehints will actually break python 3.8. So we should be aware of this issue. My recommendation is that we should start depreciating our use of python 3.8 in favour of 3.9 or 3.10.